src/bvh/util/array.hpp

```diff
@@ -35,44 +35,12 @@
 
 #include "attributes.hpp"
 
-#ifndef BVH_ENABLE_CUDA
 #include <array>
-#endif
```

Comment:
`<array>` is included without `#ifndef` in multiple other places. This seems to at least build fine.

Reply:
Excellent, and if it causes runtime errors we should use `Kokkos::Array` rather than having our own wrapper